rust/src/tests/chmod/mod.rs

```diff
@@ -3,7 +3,7 @@ mod lchmod;
 mod permission;
 
 /// Wrapper for `fchmodat(None, path, mode, FchmodatFlags::FollowSymlink)`.
-fn chmod<P: ?Sized + nix::NixPath>(path: &P, mode: nix::sys::stat::Mode) -> nix::Result<()> {
+pub fn chmod<P: ?Sized + nix::NixPath>(path: &P, mode: nix::sys::stat::Mode) -> nix::Result<()> {
```

Comment:
If this function will be used by many different tests, I suggest moving it into rust/src/tests/mod.rs instead of making it public.